backend/inspirehep/migrator/tasks.py

```diff
@@ -291,13 +291,14 @@ def create_records_from_mirror_recids(self, recids):
             raise
         except Exception:
             LOGGER.exception("Cannot migrate record", recid=recid)
-            continue
+            raise
     try:
         db.session.commit()
     except Exception:
         LOGGER.exception(
             "Cannot commit migrated records", processed_records=list(processed_records)
         )
+        raise
     models_committed.connect(index_after_commit)
```

Comment:
```suggestion
    finally:
        models_committed.connect(index_after_commit)
```

Reply:
I'm not sure if this is needed as after raising this db connection will be killed. But better safe than sorry. :+1: